.github/workflows/hqs-build-deploy-rust-pyo3-release.yml

```diff
@@ -0,0 +1,71 @@
+name: HQS build and deploy for rust pyo3 repos (release)
+
+on:
+  workflow_dispatch:
+  release:
+    types: [released]
+
+jobs:
+  api_integration:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          default: true
+          components: rustfmt
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+      - run: |
+          python -m pip install numpy
+          cargo test --workspace --no-default-features --locked
```

Comment:
We also should have a duplicate of this running with QRYD_API_HQS=1 that may not fail and this one should be allowed to fail.